src/models/contracts/objectExplorer/refreshSessionRequest.ts

```diff
@@ -0,0 +1,22 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { ExpandParams } from './expandNodeRequest';
+import { RequestType } from 'vscode-jsonrpc';
+
+/**
+ * Parameters to the ExpandRequest.
+ */
+export class RefreshParams extends ExpandParams {}
```

Comment:
>ExpandParams [](start = 35, length = 12)

I don't think this should be extending ExpandParams. They have nothing to do with each other so shouldn't be directly related. It's fine in this case to just copy the params if you want to use the same ones

Reply:
I'm just following what we have in SQL Tools Service, and ADS does the same thing. They both have the same class members so it kind of makes sense.